### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by Andrey-Pidgorodetsky**
add calendar

### CODE REVIEW

File: src/store/action/tasks-actions.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export const addTaskAction = (task) => {
+    return {
+        type: "ADD_TASK",
+        payload: task
+    };
+};
+
+export const dellTaskAction = (id) => {
```
Comment (letrylesha) [inline src/store/action/tasks-actions.js]: deleteTaskAction

File: src/components/pages/CalendarTodos/CalendarTodos.jsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  };
+
+    return (
+        <div>
+            <Input value={taskName} placeholder="Enter task name" onChange={handleTaskNameChange}/>
+            <DatePicker onChange={handleDatePiker}/>
+            <Button style={{background:"#E0FFFF"}} onClick={handlAddTask}>Add task</Button>
+            <div> { 
+            props.tasks.length >= 0 ?
+                props.tasks.map(task =>
```
Comment (letrylesha) [inline src/components/pages/CalendarTodos/CalendarTodos.jsx]: move it to separate component

TasksList

File: src/components/pages/CalendarTodos/CalendarTodos.jsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    props.dellTask(task.id)
+  };
+
+    return (
+        <div>
+            <Input value={taskName} placeholder="Enter task name" onChange={handleTaskNameChange}/>
+            <DatePicker onChange={handleDatePiker}/>
+            <Button style={{background:"#E0FFFF"}} onClick={handlAddTask}>Add task</Button>
+            <div> { 
+            props.tasks.length >= 0 ?
```
Comment (letrylesha) [inline src/components/pages/CalendarTodos/CalendarTodos.jsx]: you can leave just props.tasks.length

File: src/components/Leftblock/Buttons/Buttons.jsx [hunk 4]
```diff
@@ -19,9 +19,17 @@ const style = {
...
-        cursor: "pointer", 
+        cursor: "pointer",
+        textAlign: "left",
+    },
 
+    p: {
+        display: "flex",
+        alignItem: "center"
     },
+    pHower : {
```
Comment (letrylesha) [inline src/components/Leftblock/Buttons/Buttons.jsx]: remove

Comment (Andrey-Pidgorodetsky) [reply src/components/Leftblock/Buttons/Buttons.jsx]: Done

File: src/store/action/tasks-actions.js (hunk 1 above)
Comment (Andrey-Pidgorodetsky) [reply src/store/action/tasks-actions.js]: Done

File: src/components/pages/CalendarTodos/CalendarTodos.jsx (hunk 2 above)
Comment (Andrey-Pidgorodetsky) [reply src/components/pages/CalendarTodos/CalendarTodos.jsx]: Done

**Pull request merged**